### ISSUE TITLE
No way to return to search from slide mode

### ISSUE THREAD

**Issue by sukhpalsingh** [sikher/sikher#11]
When Slide mode is enabled for displaying the hymn, after click the pankti from the search result, the hymn is opening fine in the slide mode. But there is not way to go back to search from the slide mode.

**Comment by sukhpalsingh** [comment]
I found the go back options today but it was nearly invisible because of dim black color. It will be hard for normal user to find these options, so i recommend to make it little more white to stand this out to the user.

**Comment by jasdeepkhalsa** [comment]
Yes I agree... I think all the controls on that slides view should be one of the background tones Base2 or Base3 from http://ethanschoonover.com/solarized
